vmm/src/default_syscalls/x86_64.rs

```diff
@@ -47,6 +47,7 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
                     and![Cond::new(1, Eq, super::FUTEX_WAIT_PRIVATE)?],
                     and![Cond::new(1, Eq, super::FUTEX_WAKE_PRIVATE)?],
                     and![Cond::new(1, Eq, super::FUTEX_REQUEUE_PRIVATE)?],
+                    and![Cond::new(1, Eq, super::FUTEX_CMP_REQUEUE_PRIVATE)?],
```

Comment:
Are the changes in this file specifically for `gnu` only? Should they be conditionally compiled then?

Reply:
Fixed it.